package.json

```diff
@@ -34,7 +34,9 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@emotion/styled": "^11.10.6",
     "@mdx-js/react": "^2.3.0",
+    "@mui/icons-material": "^5.11.16",
```

Comment:
I think we were deciding in adopting `react-icons` could you switch the `@mui/icons-material` for the react-icons one?